modules/data.atmosphere/R/extract_ERA5.R

```diff
@@ -0,0 +1,90 @@
+#' ERA5_extract
+#'
+#' @param lat latitude
+#' @param long longitude
+#' @param years years to be extracted
+#' @param vars variables to be extarcted. If NULL all the variables will be returned.
+#' @param data.folder Path to the directory where ERA5 nc files are located.
+#' @details For the list of variables check out the documentation at \link{https://confluence.ecmwf.int/display/CKB/ERA5+data+documentation#ERA5datadocumentation-Spatialgrid}
+#'
+#' @return a list of xts objects with all the variables for the requested years
+#' @export
+#' @examples
+#' \dontrun{
+#' point.data <- ERA5_extract(lat=40, long=-120, years=c(1990:1995), vars=NULL, data.folder=".")
+#' 
+#  point.data %>% 
+#'  map(~xts::apply.daily(.x, mean))
+#'
+#' }
+ERA5_extract <-
+  function(lat = 40,
+           long = -120,
+           years = c(2006),
+           vars = NULL, 
+           data.folder) {
+    
+    tryCatch({
+      # for each ensemble
+      one.year.out <- c(1:10) %>%
```

Comment:
Will there always be 10 ensemble members? If so, please define this as a constant (e.g. `ensembles <- seq(1, 10)`) and then use it throughout, rather than re-using `1:10`.